app/root.tsx

```diff
@@ -29,7 +30,7 @@ export const links: LinksFunction = () => [
   },
   {
     rel: 'stylesheet',
-    href: 'https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap',
+    href: 'https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Kode+Mono:wght@400..700&family=Teko:wght@300..700&display=swap',
```

Comment:
Do we even need Inter family anymore? 

Also, I don't know a lot about fonts so might be a stupid q, would it be better to download fonts and have them served with our app?

Reply:
Yea I'll remove Inter
and I'll look into downloading vs network request before our call then we can try to decide

Reply:
actually my q was indeed stupid. even if we download and serve fonts form our server the request is still made to fetch them. the diff is that in that case it's fetched from there instead of from the google server.

I also assume that google fonts are served from cdns so we are probably good.

but still take a look in other pros and cons I might be missing